lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'takeaway'
+
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = { "Prawn Crackers" => 1.51, "Prawn Gyoza" => 3.51,
+       "Chicken Gyoza" => 3.11, "Spring Rolls" => 3.15 }
+  end
+
+  def print
```

Comment:
This would also be more readable if, by virtue of having a Dish / MenuItem class you could do things like:

```
@dishes.map do |dish|
  "#{dish.name} - #{dish.price}"
end
```